pkg/cmd/pipeline/start.go

```diff
@@ -247,14 +248,24 @@ func (opt *startOptions) getInputResources(resources resourceOptionsFilter, pipe
 
 func (opt *startOptions) getInputParams(pipeline *v1alpha1.Pipeline) error {
 	for _, param := range pipeline.Spec.Params {
-		var ans string
+		var ans, ques, defaultValue string
+		ques = fmt.Sprintf("Value of param `%s` of type `%s`?", param.Name, param.Type)
```

Comment:
```suggestion
		ques = fmt.Sprintf("Is value of param `%s` of type `%s`?", param.Name, param.Type)
```

Reply:
its a question, which asks the value of params, `Is the value` does not look good to me

Reply:
Okay, I think it reads a little clearer, but it’s a nit. If it comes up again, we can change it later.

Reply:
shouldn't it be "value **for** param …" ?

Reply:
that change I can make

Reply:
Fixed.